examples/ui-demo/src/components/modals/Erc20/Erc20Modal.tsx

```diff
@@ -47,7 +47,8 @@ export function Erc20Modal({
   const { data: ethPriceData, isLoading: isLoadingEthPrice } = useGetEthPrice();
 
   const chain = accountMode === "7702" ? baseSepolia : arbitrumSepolia;
-  const rpcUrl = chain.rpcUrls.default.http[0];
+  const rpcUrl =
+    chain.rpcUrls.alchemy?.http?.[0] ?? chain.rpcUrls.default.http[0];
```

Comment:
i think now this uses the alchemy rpc w/o an api key. not sure how quickly it will get rate limited. could we just use the same proxy'd rpc url that we are providing to the `alchemy` transport below?